src/open62541_compat.cpp

```diff
@@ -26,13 +26,10 @@
 #include <Utils.h>
 #include <boost/format.hpp>
 #include <boost/date_time.hpp>
-//#include <ASUtils.h>
 
-class alloc_error: public std::runtime_error
-{
-public:
-    alloc_error(): std::runtime_error("memory allocation exception") {}
-};
+#include <open62541_compat_common.h>
+
+
 				 
```

Comment:
Hi Piotr,

The UaStatus::IsBad() implementation is a bit or a surprise. There is some implicit string -> bool magic goes on here? Perhaps explicit would be safer in the long term for maintenance (or maybe just a comment)